### PR TITLE
Remove deprecated function Add

### PR DESCRIPTION
**Pull request by follitude**

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
Please fix conflict
rest LGTM

File: tests/test_ingest_external_file.rs [hunk 1]
```diff
@@ -86,3 +142,100 @@ fn test_ingest_external_file() {
...
+    db.ingest_external_file_cf(handle, &ingest_opt, &[test_sstfile_str])
+        .unwrap();
+
+    assert_eq!(db.get_cf(handle, b"k1").unwrap().unwrap(), b"a");
+    assert_eq!(db.get_cf(handle, b"k2").unwrap().unwrap(), b"b");
+    assert!(db.get_cf(handle, b"k3").unwrap().is_none());
+    assert_eq!(snap.get_cf(handle, b"k1").unwrap().unwrap(), b"a");
+    assert_eq!(snap.get_cf(handle, b"k2").unwrap().unwrap(), b"b");
+    assert_eq!(snap.get_cf(handle, b"k3").unwrap().unwrap(), b"c");
+}
```
Comment (andelf) [inline tests/test_ingest_external_file.rs]: Eof

**Comment by siddontang** [comment]
CI failed @follitude

**Comment by siddontang** [comment]
Please fix the conflict.

**Pull request closed without merging**